src/test/java/tools/jackson/databind/ser/jdk/WriteDatesAsTimestamps5142JdkTests.java

```diff
@@ -0,0 +1,63 @@
+package tools.jackson.databind.ser.jdk;
+
+import java.util.*;
+
+import org.junit.jupiter.api.Test;
+import tools.jackson.databind.ObjectMapper;
+import tools.jackson.databind.cfg.DateTimeFeature;
+import tools.jackson.databind.ext.javatime.DateTimeTestBase;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+// [databind#5142] JSTEP-5 Tests to verify current behavior and discuss further action DateTimeFeature.WRITE_DATES_AS_TIMESTAMPS
+public class WriteDatesAsTimestamps5142JdkTests
+    extends DateTimeTestBase
+{
+    private static final ObjectMapper WITH_TIMESTAMP_MAPPER = mapperBuilder()
+            .enable(DateTimeFeature.WRITE_DATES_AS_TIMESTAMPS)
+            .build();
+
+    private static final ObjectMapper WITHOUT_TIMESTAMP_MAPPER = mapperBuilder()
+            .disable(DateTimeFeature.WRITE_DATES_AS_TIMESTAMPS)
+            .build();
+
+    @Test
+    public void testWriteDatesAsTimeStamps()
+        throws Exception
+    {
+        // java.util.Date
+        _testTimestamp(
+                new Date(1234567890123L),
+                Date.class,
+                // Expected... [2009,1,13,23,31,30,123],
```

Comment:
Timestamp is a simple number so I don't think we'd want JSON Array... ?

Although perhaps with `@JsonFormat(shape = Shape.ARRAY)` could force such output.